### PR TITLE
feat: increase timeout for getPin lambdas

### PR DESCRIPTION
**Pull request by ericssontadashi**
Increase timeout for getPin lambdas
Added timeout for remaining lambdas

### CODE REVIEW

**Comment by ericssontadashi** [comment]
image

File: api/basic/get-pin-router.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export async function handler (event: APIGatewayProxyEventV2, context: Context):
 
   logger.info('Get pin request')
 
+  await delay(40000)
```
Comment (codeflyer) [inline api/basic/get-pin-router.ts:33]: What's this for?

File: stacks/BasicApiStack.ts [hunk 2]
```diff
@@ -118,19 +118,22 @@ export function BasicApiStack ({ app, stack }: StackContext): { queue: Queue, bu
       'GET    /internal/pins/{cid}': {
         function: {
           handler: 'basic/get-pin.handler',
-          functionName: formatResourceName(app.stage, 'getPin')
+          functionName: formatResourceName(app.stage, 'getPin'),
```
Comment (codeflyer) [inline stacks/BasicApiStack.ts:121]: Why do we need a timeout for the internal lambda as well? Do we have any timeout on them?

File: api/basic/get-pin-router.ts (hunk 1 above)
Comment (ericssontadashi) [reply api/basic/get-pin-router.ts:33]: Timeout to validate apigw behavior, gw has a hard limit of 30s.
This pr it's for test purpose only

Comment (codeflyer) [reply api/basic/get-pin-router.ts:33]: ok, clear.

**Pull request closed without merging**